docs/dapp/opl/celer/networks.md

```diff
@@ -40,7 +40,6 @@ description: Supported networks on Celer IM
 | Filecoin Hyperspace Testnet | 3141 | 0xc45 | filecoin-testnet |
 | FNCY Testnet | 923018 | 0xe158a | fncy-testnet |
 | Godwoken Testnet | 71401 | 0x116e9 | godwoken-testnet |
-| Goerli Testnet | 5 | 0x5 | goerli |
 | Polygon Mumbai Testnet | 80001 | 0x13881 | polygon-mumbai |
```

Comment:
```suggestion
```
all these testnets don't exist anymore additional to goerli:
arbitrum
consensys
filecoin hyperspace
godwoken
polygon mumbai
polygon zkevm

Reply:
Godwoken is still okay? 🤔 
https://v1.testnet.gwscan.com/

Removed others.